# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../libspice/sasl2.patch
# Copyright (C) 2016 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

--- spice-0.12.7/server/reds.c.vanilla	2016-04-24 15:34:02.766270012 +0000
+++ spice-0.12.7/server/reds.c	2016-04-24 15:35:29.114273799 +0000
@@ -40,7 +40,7 @@
 #include <openssl/err.h>
 
 #if HAVE_SASL
-#include <sasl/sasl.h>
+#include <sasl2/sasl.h>
 #endif
 
 #include <glib.h>
--- spice-0.12.7/server/reds_stream.c.vanilla	2016-04-24 15:34:36.442271489 +0000
+++ spice-0.12.7/server/reds_stream.c	2016-04-24 15:35:20.322273413 +0000
@@ -46,7 +46,7 @@
 extern SpiceCoreInterface *core;
 
 #if HAVE_SASL
-#include <sasl/sasl.h>
+#include <sasl2/sasl.h>
 
 typedef struct RedsSASL {
     sasl_conn_t *conn;
